draft-irtf-cfrg-vdaf.md

```diff
@@ -1720,7 +1726,10 @@ class PrgFixedKeyAes128(Prg):
         #
         # Implementation note: This step can be cached across PRG
         # evaluations with many different seeds.
-        self.fixed_key = cSHAKE128(binder, 16, b'', dst)
+        # The length of dst, the domain separation string, should be <256 bytes
+        # A longer dst will cause an OverflowError
+        dst_length = len(dst).to_bytes(1, byteorder = 'little') #one byte value
+        self.fixed_key = SHAKE128(dst_length + dst + binder, 16)
```

Comment:
(For consideration in a follow-up PR.) @schoppmp We might go with plain SHA-3 in order instead of SHAKE128 since the output length is fixed.